src/Aspire.Hosting.Azure/AzureBicepResource.cs

```diff
@@ -134,11 +139,18 @@ public virtual string GetBicepTemplateString()
     /// <param name="context">The <see cref="ManifestPublishingContext"/>.</param>
     public virtual void WriteToManifest(ManifestPublishingContext context)
     {
-        context.Writer.WriteString("type", "azure.bicep.v0");
-
         using var template = GetBicepTemplateFile(Path.GetDirectoryName(context.ManifestPath), deleteTemporaryFileOnDispose: false);
         var path = template.Path;
 
+        if (Scope.Count == 0)
+        {
+            context.Writer.WriteString("type", "azure.bicep.v0");
+        }
+        else
+        {
+            context.Writer.WriteString("type", "azure.bicep.v1");
```

Comment:
We should probably update our json schema as well for the new values.

Reply:
@davidfowl - thoughts? Are we keeping the json schema up to date?

Reply:
I've updated the schema in https://github.com/dotnet/aspire/pull/7249/commits/48e1498be3241f03323355904648311ee75b6080. Let me know if the delta looks good.